src/components/InfoViewerSkeleton/InfoViewerSkeleton.tsx

```diff
@@ -13,22 +13,18 @@ const SkeletonLabel = () => (
     </div>
 );
 
-interface ClusterInfoSkeletonProps {
+interface InfoViewerSkeletonProps {
     className?: string;
     rows?: number;
 }
 
-export const ClusterInfoSkeleton = ({rows = 8, className}: ClusterInfoSkeletonProps) => (
+export const InfoViewerSkeleton = ({rows = 8, className}: InfoViewerSkeletonProps) => (
     <div className={b(null, className)}>
         {[...new Array(rows)].map((_, index) => (
             <div className={b('row')} key={`skeleton-row-${index}`}>
                 <SkeletonLabel />
                 <Skeleton className={b('value')} />
             </div>
         ))}
-        <div className={b('row')} key="versions">
-            <SkeletonLabel />
-            <Skeleton className={b('versions')} />
-        </div>
```

Comment:
Since `ClusterInfo` is moved to a separate tab, layout shifts on data load doesn't affect us as  much as before, so there is no need to fully preserve content height.

Removed separate versions row to make component usable for all tabs and pages